packages/app/backend-http-client/index.ts

```diff
@@ -8,14 +16,9 @@ export {
   sendPostBinary,
   httpClient,
   buildClient,
-  type RequestOptions,
-  type Response,
-  type HttpRequestContext,
-  type ResponseSchema,
-  TEST_OPTIONS,
-  JSON_HEADERS,
-  NO_CONTENT_RESPONSE_SCHEMA,
-  UNKNOWN_RESPONSE_SCHEMA,
```

Comment:
Removed:
- `Response` -> looks like it was not being used, as we are using RequestResultt from `undici-retry`
- `ResponseSchema` -> using `ZodSchema` instead
- `TEST_OPTIONS` -> was only used on internal tests
- `NO_CONTENT_RESPONSE_SCHEMA` -> was only used on internal tests

Please let me know if you disagree with one of this changes 🙏

Reply:
TEST_OPTIONS and NO_CONTENT_RESPONSE_SCHEMA are exposed so that other services could use them for their tests as well, we actually use at least TEST_OPTIONS in e. g. node-service-template

Reply:
Ok, will put `TEST_OPTIONS` back thanks for the clarification 🙏 

About `NO_CONTENT_RESPONSE_SCHEMA`, does it make sense to have it? `204` will now return `null` and there is no validation

Reply:
but http client still requires you to pass some schema. and we can't make that requirement conditional, because in some cases we are only going to get 204 (e. g. DELETE), but in some cases we may get either 200 or 204 (GET)

Reply:
You are right, but I think it is something that can be done by the user, but I have no strong opinion/problem with having it, so adding it back

Reply:
it can be, but it's slightly more convenient not to have to reinvent it on every service :D

Reply:
Yeah, sounds like a good reason, thanks for mentioning it Igor 🙏, both are back already

Reply:
curious if removing Response will break something or not. let's try removing it and see if we miss it :D